modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -202,15 +211,21 @@ spec:
         address:
         - ip: "fd69::2"
         prefix-length: 125
+# ...
 ----
 <1> Name of the policy.
 <2> Name of the interface.
 <3> The type of ethernet.
 <4> The requested state for the interface after creation.
 <5> Disables IPv4 and IPv6 in this example.
 <6> The node NIC to which the bridge is attached.
+<7> Set to `48` so that the default Dynamic Host Configuration Protocol (DHCP) route, which is automatically enabled on new network interfaces, does not take precedence over the `br-ex` route.
```

Comment:
looks like we need to sync the note with above.

Reply:
:heavy_check_mark: